contentcuration/contentcuration/frontend/shared/vuex/file/actions.js

```diff
@@ -281,3 +281,29 @@ export function getAudioData(context, url) {
       .catch(reject);
   });
 }
+
+export function downloadFile(context, { url, fileName }) {
+  return new Promise((resolve, reject) => {
```

Comment:
I have some other thoughts on whether using the axios client is worthwhile (see my other comment), but generally speaking:

Wrapping this in a Promise is unnecessary. The client used inside of the promise constructor already returns a promise, and so there is no difference between returning the resulting promise and creating a promise that resolves after the chained logic. Either way, you can also remove the try/catch because that would be the default behavior for a callback.